src/preprocess.py

```diff
@@ -248,7 +249,7 @@ def build_tasks(args):
         indexers["words"] = SingleIdTokenIndexer()
     if args.elmo:
         indexers["elmo"] = ELMoTokenCharactersIndexer("elmo")
-        assert args.tokenizer in {"", "MosesTokenizer"}
+        assert args.tokenizer in tokenizers.AVAILABLE_TOKENIZERS
```

Comment:
We need to keep the emptystring case (`args.tokenizer == ""`) - per correspondence with ELMo authors, ELMo doesn't expect Moses tokenization so the default should be to just split on spaces. 

(I left `MosesTokenizer` as an option to not break other experiments, but we should actually prefer emptystring and have a warning if trying to use Moses with ELMo).